src/main/java/com/dev/cinema/dto/user/UserRequestDto.java

```diff
@@ -1,9 +1,11 @@
 package com.dev.cinema.dto.user;
 
+import javax.validation.constraints.Email;
 import lombok.Data;
 
 @Data
 public class UserRequestDto {
```

Comment:
do we use this dto anywhere?

Reply:
Not at the moment, but in the future will do probably, so I'll remove it.